XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs

```diff
@@ -1,65 +1,74 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
+#if WINDOWS_UWP
+using Windows.Foundation.Metadata;
+#elif (UNITY_WSA && DOTNETWINRT_PRESENT)
```

Comment:
We're not using the dot net scripting backend so this doesn't apply to the xrtk

Reply:
Yes, this can go.